docs/src/index.md

```diff
@@ -9,13 +9,10 @@ First, prepare Julia environment v >= 1.7.0.
 Second, add the package in Pkg mode
 ```julia
 julia>]add NuclearToolkit
-``` 
-Note: The above is currently not working, since this package has not yet registered as an official package.  
-The adhoc prescription for now is to execute below
-```bash
-julia src/package_install.jl
```

Comment:
I was about to point this was out of date! Glad you got to it.
You may be aware that pkg3 allows installing from github:

```julia
] add https://github.com/SotaYoshida/NuclearToolkit.jl
```

So the install file was never truly necessary.